portality/templates-v2/management/base.html

```diff
@@ -71,12 +74,10 @@ <h2 class="sr-only">DOAJ Dashboard</h2>
         </nav>
         {% block nav %}{% endblock %}
     </header>
-
-
     <main>
-        <header class="main-header">
+        <header class="main-header {{ self.fixed_header() }}">
```

Comment:
I think we can put `{% block management_main_header_classes %}{% endblock %}` here directly.  Comparing this with, for example, `body_class` on the root `base.html` which specifies this where it's used, and then sub-templates override it.

Reply:
I find it easier to figure out what I need in a child's template if the blocks are defined at the top and then just called. It also theoretically is more robust and safer (although this is a simple case, so both are equivalent).